src/cim/components/featureSupportLevels/ACMFeatureSupportLevelProvider.tsx

```diff
@@ -24,32 +45,36 @@ export const ACMFeatureSupportLevelProvider: React.FC<ACMFeatureSupportLevelProv
   const ocpVersions = React.useMemo(() => getOCPVersions(clusterImages), [clusterImages]);
 
   const supportLevelData: FeatureSupportLevelsMap = React.useMemo<FeatureSupportLevelsMap>(() => {
-    const featureSupportLevelsMap: FeatureSupportLevelsMap = {};
+    return getFeatureSupportLevelsMap();
+  }, []);
 
-    ocpVersions.forEach((version) => {
-      // TODO(mlibra): get this dynamically
-      const featureIdToSupportLevel: FeatureIdToSupportLevel = {
-        REQUESTED_HOSTNAME: 'supported',
-        SNO: 'tech-preview',
-        CNV: 'supported',
-        NETWORK_TYPE_SELECTION: 'tech-preview',
-      };
-      featureSupportLevelsMap[version.version] = featureIdToSupportLevel;
-    });
-    return featureSupportLevelsMap;
-  }, [ocpVersions]);
+  const getVersion = (version = '') => {
+    const match = /[0-9].[0-9][0-9]?/g.exec(version);
+    if (match && match[0]) {
+      return match[0];
+    }
+    return '';
+  };
+
+  const getNormalizedVersion = React.useCallback(
+    (versionName: string) => {
+      const version = ocpVersions.find((v) => v.value == versionName);
```

Comment:
there might be cases where the version doesn't match exactly with ocpVersions. you can remove setting ocpVersions all together. just find the version in featureSupportLevel that is a prefix of the versionName

Reply:
@batzionb @jtomasek in OpenShift version dropdown are shown the list of OpenShift versions that have clusters that are being installed before. Is that OK? Or we have to put the list of supported versions?
![image](https://user-images.githubusercontent.com/11390125/158533675-4da77cd7-9f8c-42c9-b32d-5f13b7ec9962.png)

Reply:
it could be a user installed a cluster with a different version than the one in the list, from a previous ACM release. for example 4.8.2. He should still be able to open the cluster and see that SNO is dev preview. 
To find the support level, you need to to take the x.y part of the version and find it in the feature support level object. you don't even need to do the parsing just find the version from the feature support level object that is the prefix of the cluster version